packages/cozy-app-publish/package.json

```diff
@@ -6,9 +6,7 @@
   "license": "MIT",
   "scripts": {
     "test": "jest --verbose --coverage",
-    "test:manual": "jest test/manual.spec.js",
-    "test:publish": "jest test/publish.spec.js",
-    "test:travis": "jest test/travis.spec.js"
```

Comment:
I did not know why there was that ? It does not seem so useful to me.

Reply:
We was using it to test separately by we don't use them anymore, we run all tests on travis here